### ISSUE TITLE
How to download Firefox 40.0 32-bit Win32?

### ISSUE THREAD

**Issue by juj** [mozilla/mozdownload#333]
I'm trying to verify if Firefox 40.0 had a bug that could have been fixed in Firefox 41.0, so I'm attempting to get FF 40.0 Win32 as a zip package, with the following:

    mozdownload --type=release --version=40.0 --platform=win32 --extension=zip

or

    mozdownload --type=release --version=40.0 --platform=win32

but both of those give

    mozdownload.errors.NotFoundError: Specified build has not been found: https://archive.mozilla.org/pub/mozilla.org/firefox/releases/40.0/win32/en-US

Unfortunately from the error message, I'm at a dead end to what I'm doing wrong. As a workaround, I'm uninstalling Firefox, and installing manually from installer, but it would be nice to be able to download a .zip file of a release via mozdownload and unzip it to a temp directory for quick testing. If this is possible, perhaps it might be something to add to the example command line invocations?

**Comment by parkouss** [comment]
Hey Juj, you can try mozregression (http://mozilla.github.io/mozregression/) for your use case.

for example,
```
mozregression --launch 2015-05-11 # download and run the build for release 40
mozregression --launch 2015-06-29 # download and run the build for release 41
```

We are working on allowing the release number (https://bugzilla.mozilla.org/show_bug.cgi?id=1214192) so you could do:
```
mozregression --launch 40  # or 41
```
But this is not ready yet, should be released soon (worse next week).

**Comment by parkouss** [comment]
Oh, please specify "--bits=32" if you want 32 bit build and that you are running on a 64 bit os.

**Comment by juj** [comment]
Thanks! Using mozregression now, that got me going.

**Comment by whimboo** [comment]
There was a migration happening from the FTP server to S3. That's why lots of builds were not accessible. With the newly released mozdownload 1.19 all should work fine again.
